changelogs/fragments/1900-zos_job_query-map_system_subsystem.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - zos_job_query - system and subsystem values are captured directly from zoau response and mapped in the modules response.
```

I would rephrase this since is a customer facing change.
```suggestion
  - zos_job_query - System and Subsystem are now retrieved from JES.
```

---

You can review how to write a changelog fragement and its guidelines here. https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment